corehq/apps/locations/fixtures.py

```diff
@@ -142,7 +163,7 @@ def _get_metadata_node(location):
 
 
 def _location_to_fixture(location_db, location, type_lookup_function):
-    root = Element(type_lookup_function(location.location_type), {'id': location._id})
+    root = Element(type_lookup_function(location.location_type.name), {'id': location.location_id})
```

Comment:
I think `type_lookup_function(location.location_type.name)` is the equivalent of `location.location_type.code` right? Can probably do away with that abstraction entirely now that we have proper `LocationType` objects.

Though... for performance reasons you may want to do a `select_related` on the original query to avoid potentially hitting the database a second time for each location.